src/bin/ipa_bench/cmd.rs

```diff
@@ -158,15 +146,12 @@ impl Command {
         let sample = Sample::new(&config);
 
         let mut rng = random_seed.map_or(StdRng::from_entropy(), StdRng::seed_from_u64);
-        let mut ss_rng = random_seed.map_or(StdRng::from_entropy(), StdRng::seed_from_u64);
 
         let (s_count, t_count) = generate_events(
             &sample,
             DEFAULT_EVENT_GEN_COUNT * scale_factor,
-            epoch,
-            secret_share,
+            &epoch,
```

Comment:
why are `sample` and `epoch` passed in as shared values?

Reply:
I had the same Clippy warning and my conclusion is that we should suppress it because it makes our API look weird for arguments that implement `Copy`